zebra-state/src/lib.rs

```diff
@@ -26,7 +26,6 @@ use zebra_chain::{
     parameters::Network,
 };
 
-pub mod in_memory;
 pub mod on_disk;
```

Comment:
I don't think this module should be part of the public API any more, since there's no in-memory/on-disk distinction.

Reply:
we have a follow up issue for this already

Reply:
Yep, see edit/clarification here: https://github.com/ZcashFoundation/zebra/pull/974#pullrequestreview-480048425  

This PR already changes all the import paths, so it might as well change them to the correct ones.  Editing the internal structure of the code can be done later with no problems, once the import paths are correct.